packages/client/src/components/ui/button/button.tsx

```diff
@@ -6,14 +6,16 @@ type ButtonProps = {
     buttonSize?: 'small' | 'medium' | 'large';
     buttonStyle?: 'normal' | 'outlined';
     onClick?: MouseEventHandler;
+    type?: 'button' | 'submit' | 'reset' | undefined;
```

Comment:
Сам оператор optional уже добавляет значение undefined.
```suggestion
    type?: 'button' | 'submit' | 'reset';
```

Reply:
Понял, спасибо